experimental/check_gpu_ecc/check_gpu_ecc.py

```diff
@@ -9,11 +9,16 @@
 import socket
 import json
 import csv
+from datetime import timedelta
+from datetime import datetime
 from urllib.request import urlopen, Request
 
 
 ECC_COUNTER_THRESHOLD = 20000000
-SRAM_ECC_COUNTER_THRESHOLD = 10000
+SRAM_ECC_COUNTER_THRESHOLD = 4
+RETIRED_PAGES_THRESHOLD = 62
+RETIRED_PAGES_30D_THRESHOLD = 5
+RP_DAYS = 30
```

Comment:
RP_DAYS differs from the RETIRED_PAGES_* pattern, why?

Reply:
Thats in the doc you shared.? https://eng.ms/docs/cloud-ai-platform/azure-core/azure-specialized/hpcai/azure-hpc/azure-hpc-tsg/gpu/pageretirement#rma-criteria